editor/script_editor_debugger.cpp

```diff
@@ -2336,4 +2356,4 @@ ScriptEditorDebugger::~ScriptEditorDebugger() {
 
 	server->stop();
 	_clear_remote_objects();
-}
+}
```

Comment:
Missing `\n` character at the end of the file.

Reply:
fixed